syntax = "proto3";

message Hero {
	int32 id = 1;
	string name = 2;
	int32 strength = 3;
	int32 resistance = 4;
	int32 health = 5;
}

message HeroID {
	int32 id = 1;
}